app/main/menu.js

```diff
@@ -368,7 +368,7 @@ class AppMenu {
 
 	static sendAction(action, ...params) {
 		const win = BrowserWindow.getAllWindows()[0];
-
+	
```

Comment:
@abhigyank any reason to add this, linter fails locally?

Reply:
That was mistake. Sorry!
But it didn't fail lint test while commiting though.

Reply:
That's weird, no problem anymore since got the CI working again so it fails on linting error which it should have done.